test/test_backend/test_backend.py

```diff
@@ -339,134 +339,3 @@ def test_get_cset_members_items__cols(self):
             {'concept_id': 4091006, 'standard_concept': 'S', 'vocabulary_id': 'SNOMED'},
             {'concept_id': 4052321, 'standard_concept': 'S', 'vocabulary_id': 'SNOMED'}].sort(key=key)
         self.assertEquals(csmi, expected)
-
-    # TODO: test is failing. fix
-    #  - The cause of the err is that subgraph() is a different function now. it used to call connected_subgraph_from_nodes(), which no longer exists.
-    #    Is this test still valid? Is there any func that takes a list of concept IDs and returns their parentage tuples?
-    @unittest.skip("May need to remove this test. I think the code it was testing got removed. Skipping failing/erroring test temporarily.")
-    def test_subgraph(self):
```

Comment:
## Moved `test_subgraph()`
To `test_graph.py`. Still doesn't work. That's a separate issue.